source/module_hamilt_lcao/module_deepks/LCAO_deepks.cpp

```diff
@@ -258,13 +268,23 @@ void LCAO_Deepks::init_gdmepsl()
 {
     this->gdm_epsl = new double** [6];
     
+    int pdm_size;
+    if(!if_equiv)
+    {
+        pdm_size = (this->lmaxd * 2 + 1) * (this->lmaxd * 2 + 1);
+    }
+    else
+    {
+        pdm_size = this -> des_per_atom;
```

Comment:
two extra spaces appear before and after ->

Reply:
what is the criteria here?

Reply:
It is a suggsetion that you can use clang-format